crates/pallet-feeds/src/lib.rs

```diff
@@ -143,21 +131,21 @@ mod pallet {
         #[pallet::weight(10_000)]
         pub fn create(
             origin: OriginFor<T>,
+            feed_id: T::FeedId,
```

Comment:
No need to block anything. The way it was before with non-repeating incremental IDs worked well. Also deletion wasn't really immediate priority, but it you already implemented it - sure, why not.

Reply:
I'm not sure if having a close/delete feed make sense TBH. 
We would need to guard something who can delete/close the feed. we need to introduce the ownership of the feed.

Simplest idea would be to make close/delete permissioned call and let the feed owner decide to close/delete it. It would be more simpler if we let them have that decision no?

Reply:
ah cool. then that change would be a follow to this one. I would still like to keep the FeedId coming from the externality and let the owner decide to delete and release the feed if it required to

Reply:
I think it is confusing to have one feed in place of another deleted one. What good use case is when you want to let someone random override existing feed with other content (literally mess up previously working object mappings).

Reply:
only if the owner deletes it 😉 
for things we dont want to override, we just close the feed no?

Reply:
Deletion is useful because creator will get a refund for the data they stored in feed state. Why do you want so bad for people to guess which feed ID is already used and which isn't?

This whole discussion is why deletion and closing was in advanced features and not necessarily being implemented right away, definitely not in the very next PR :confused:

Reply:
> Why do you want so bad for people to guess which feed ID is already used and which isn't?

i like the choice of deleting by the owner. We can revert to incrementing one like before :)

Reply:
I don't mind generic parameter, but let's bring auto-incrementing back for now